modules/db/main.tf

```diff
@@ -19,3 +19,19 @@ module "postgres" {
 
   depends_on = [module.db_namespace]
 }
+
+module "db_init" {
+  source = "./db-init"
+  count  = var.deploy_db_init == true ? 1 : 0
+
+  namespace                                = var.namespace
+  db_init_version                          = var.db_init_version
+  postgres_additional_configuration_values = var.postgres_additional_configuration_values
+  custom_postgres_values_yaml              = var.custom_postgres_values_yaml
+  custom_input_map                         = var.custom_input_map
```

Comment:
can both the `db_init` module and `postgres` module use the same `custom_input_map` settings? 

I noticed that the custom_input_map contains a key of `postgres.deployment.schemaVersionTable = swoop.schema_version` that actually appears at postgres.service.schemaVersionTable in the postgres helm chart and doesn't appear at all in the db-init helm chart. If a value is provided via Terraform but it does not appear in the values.yaml for that respective helm chart, is it automatically inserted at that place? (just more of a question)